Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1110,9 +1110,16 @@ class BrowserViewController: UIViewController {
         }
     }
 
-    func openBlankNewTab(isPrivate: Bool = false) {
+    func openBlankNewTab(focusLocationField: Bool, isPrivate: Bool = false) {
         popToBVC()
         openURLInNewTab(nil, isPrivate: isPrivate, isPrivileged: true)
+        
+        if focusLocationField {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.3) {
+                // Without a delay, the text field fails to become first responder
```

Comment:
This delay is ugly I realize :(. But a simple .async() doesn't work, I assume I am either fighting something else setting the first responder, or simply that multiple loops of the main event loop is required for the UI state to be ready to accept a call setting the first responder.